packages/itwin/tree-widget/src/components/trees/repositories-tree/FormatLabel.tsx

```diff
@@ -0,0 +1,36 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import { RootNodeClassnameEnum } from "./GetIcon";
+
+/**
+ * @internal
+ */
+export function formatLabel(Classname: string) {
```

Comment:
renamed

Reply:
I think the `switch` in this function clearly demonstrates why a string union for `ITwinRepositoryType` would be better. Instead of

```ts
export function formatLabel(repositoryClassname: string) {
  switch (ITwinRepositoryType[repositoryClassname as keyof typeof ITwinRepositoryType]) {
    // ...
  }
}
```

You'd have

```ts
export function formatLabel(repositoryType: ITwinRepositoryType) {
  switch (repositoryType) {
    // ...
  }
}
```

Reply:
changed